core/src/test/java/com/radixdlt/consensus/liveness/MultiFactorPacemakerTimeoutCalculatorTest.java

```diff
@@ -96,15 +97,45 @@ public void timeout_should_grow_exponentially() {
             5L, 32000L);
 
     expectedTimeouts.forEach(
-        (uncommittedRounds, expectedResult) ->
+        (timeoutOccurrences, expectedResult) ->
             assertEquals(
-                expectedResult.longValue(), calculator.calculateTimeoutMs(uncommittedRounds)));
+                expectedResult.longValue(), calculator.calculateTimeoutMs(timeoutOccurrences, 0)));
+  }
+
+  @Test
+  public void timeout_should_grow_linearly_when_vertex_store_size_is_above_threshold() {
+    final var baseTimeout = 1000L;
+    // No exponent, threshold = 0.6, max multiplier = 10
+    final MultiFactorPacemakerTimeoutCalculator calculator =
+        new MultiFactorPacemakerTimeoutCalculator(
+            new PacemakerTimeoutCalculatorConfig(baseTimeout, 2.0, 0, 0L, 0.6, 10));
+
+    // spotless:off
+    final Map<Double, Long> testCases =
+        Map.of(
+          (double) -1, baseTimeout,
+          0.5, baseTimeout,
+          0.6, baseTimeout,
+          0.61, 1225L,
```

Comment:
It's slightly weird to me that half of these are in terms of baseTimeout and half of them are absolute numbers - can we standardize?